sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber1024-90s/symmetric-shake.c

```diff
@@ -1,60 +0,0 @@
-#include <stddef.h>
```

Comment:
Sean, I'd prefer not to delete anything. These folders are an exact copy of the "Reference Implementation" folder from NIST.  The only files modified to date are build files. Is this file in the way? Do we need to exclude it from the build?

Reply:
I can add it back if we want, but the Makefile doesn't use it at all, so it serves no purpose.

Reply:
I would like to leave all of the sources untouched if possible - as if we were downloading/pulling the code from the net at build time (like we do with libOQS).